src/anim/evaluator/anim-evaluator.js

```diff
@@ -456,18 +335,27 @@ class AnimEvaluator {
 
         // apply result to anim targets
         var targets = this._targets;
+        const targetValue = new Array(4);
         for (var path in targets) {
             if (targets.hasOwnProperty(path)) {
                 var target = targets[path];
-                if (this._binder.animComponent) {
+                // if this evaluator is associated with an anim component then we should blend the result of this evaluator with all other anim layer's evaluators
+                if (this._binder.animComponent && target.target.isTransform) {
```

Comment:
you could have a local variable binder as you access this multiple times
`this._binder`